code-challenges.js

```diff
@@ -20,25 +20,58 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
 
-
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// Pseudo code:
+// Pseudo code: 
+//  Combine the 2 provided variables
+//  Get the the length of the new array
+//  Return Expected output of 9
+
+// Code:
+var comboRun = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(comboRun.length)
+//  Output ----> 9
 
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
 const currentCohort = "LEARN 2023"
 // Expected output: "3202 NRAEL"
 
 // Pseudo code:
+// create variable and utilize .split on currentCohort
+// Reverse variable
+// Console.log with .join on variable
+// Return Expected Output of "3202 NRAEL"
+
+// Code:
+var reverseArray = currentCohort.split("")
+reverseArray.reverse()
+console.log(reverseArray.join(""))
+// Output ----> "3202 NRAEL"
```

Comment:
Same thing here. This is perfectly valid. I also like to have one action per line so I can log each step and see what is happening. My refactor would be 
`currentCohort.split("").reverse().join("")`